From 75045a7343c73c382e68c90a91c48b9710b702a5 Mon Sep 17 00:00:00 2001
From: Ahmed Mansour <ahmed.mansour@nxp.com>
Date: Wed, 8 Nov 2017 16:13:08 -0500
Subject: [PATCH 193/211] fsl-qbman: Include missing function prototype

Included fsl_qbman.h which defines the prototype for a function used
in arch/arm/cpu/armv8/fsl-layerscape/cpu.c for device tree fixups

Signed-off-by: Ahmed Mansour <ahmed.mansour@nxp.com>
---
 arch/arm/cpu/armv8/fsl-layerscape/cpu.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/cpu/armv8/fsl-layerscape/cpu.c b/arch/arm/cpu/armv8/fsl-layerscape/cpu.c
index 0997fcc..9d2fe1f 100644
--- a/arch/arm/cpu/armv8/fsl-layerscape/cpu.c
+++ b/arch/arm/cpu/armv8/fsl-layerscape/cpu.c
@@ -33,6 +33,7 @@
 #endif
 #include <asm/arch/clock.h>
 #include <hwconfig.h>
+#include <fsl_qbman.h>
 
 DECLARE_GLOBAL_DATA_PTR;
 
-- 
1.7.1

